pkg/controller/endpoint/endpoints_controller_test.go

```diff
@@ -111,6 +115,13 @@ func addPods(store cache.Store, namespace string, nPods int, nPorts int, nNotRea
 	}
 }
 
+func addBadIPPod(store cache.Store, namespace string, ipFamilies []v1.IPFamily) {
+	pod := testPod(namespace, 0, 1, true, ipFamilies)
+	pod.Status.PodIPs[0].IP = "0" + pod.Status.PodIPs[0].IP
+	pod.Status.PodIP = pod.Status.PodIPs[0].IP
+	store.Add(pod)
```

Comment:
```suggestion
	store.Add(pod) // nolint:errcheck
```

Reply:
or pass down the t testing.T and make the test fail, I would prefer this option

Reply:
yeah, there are 7000 other unchecked uses of store.Add() in that unit test and it felt wrong to fix just this one so I decided to ignore the linter.

Reply:
that is fair, I think the error was added recently so all previous code didn't assert on it 🤷